osu.ElasticIndexer/Commands/Index/ListIndicesCommand.cs

```diff
@@ -30,7 +30,7 @@ public virtual int OnExecute(CancellationToken token)
             Console.WriteLine($"Current schema: {currentSchema}");
             Console.WriteLine();
 
-            var indices = ElasticClient.Indices.Get(Indices.All).Indices;
+            var indices = ElasticClient.GetIndices($"{ElasticClient.AliasName}_*");
```

Comment:
I'm going to assume this is meant to work with `GetIndices` on from #152 since the current one already appends `_*`

Reply:
Yeah pretty much.